tls/extensions/s2n_client_renegotiation_info.c

```diff
@@ -113,6 +101,94 @@ static int s2n_client_renegotiation_recv(struct s2n_connection *conn, struct s2n
     return S2N_SUCCESS;
 }
 
+static int s2n_client_renegotiation_recv_if_renegotiation(struct s2n_connection *conn, struct s2n_stuffer *extension)
```

Comment:
Hmm this is not really an improvement, I was hoping we could get away from the double renegotiation. What about "s2n_client_renegotiation_recv_verify_data" and "s2n_client_renegotiation_recv_initial"
If you hate that too I'm okay with approving.

Reply:
I don't love the double renegotiation either, but I'm a little worried that if we don't match the "initial" and "renegotiation" distinctions from the RFC, it'll be confusing. 

...But I'm also now suddenly realizing I'm missing "info" there. Do you hate "s2n_client_renegotiation_info_recv_if_renegotiation" any less?

Reply:
After offline discussion, we're just going to revert the names.